docs/en/advanced/Performance-Tuning.md

```diff
@@ -277,6 +278,8 @@ alluxio.user.file.persist.on.rename=true
 # Determines the number of copies in Alluxio when files are not yet persisted, increase this to
 # a larger number to ensure fault tolerance in case of Alluxio worker failures
 alluxio.user.file.replication.durable=1
+# blacklist files which have _temporary in their path
```

Comment:
can we list some examples that would match this? At first glance I wonder 

(using glob syntax)
- Is it the file extension? `**/*._temporary`?
- Is it a path component? `**/_temporary/**`?
- Is it a directory? `**/_temporary/*`?
- Is it the filename? `**/_temporary`?
- Is it anywhere in the path? `**_temporary**`?

Reply:
We detail some examples later on, I thought it would be too wordy to put into a comment

Reply:
yeah I noticed later on. Maybe it's just me, but if I wrote this I probably would have 

```
blacklist files which contain the string "_temporary" anywhere in their path
```

Though that's mainly because when I write things I tend to over-specify sometimes - I'll leave any change up to you